app/models/user.rb

```diff
@@ -171,6 +171,14 @@ def self.recent_orders_created_for(user_id)
       .order("orders.id DESC").limit(5)
   end
 
+  def self.search_by_role(name)
```

Comment:
I don't think it's good to keep 'Charity' as a valid role input. Rather we should filter on users with organisations with a query param which indicates so.

But since we want to fix this asap, this will work fine. Can refactor later :)

Reply:
agreed, have been working with @bharat619 on a simliar solution. We'll release this as a hotfix and then a follow-up ticket to use `organisations_users` state filter properly